src/twix.coffee

```diff
@@ -515,3 +515,6 @@ if  typeof(define) == "function"
   define "twix", ["moment"], (moment) -> makeTwix(moment)
 
 @Twix = makeTwix(@moment) if @moment?
+
+# Also checks globals (Meteor)
+@Twix = makeTwix(moment) if moment?
```

Comment:
If that's fine on your side, I'll test it as a Meteor package and let you know if that's fine there too.